# PointAndAngle.msg

float64 x # [m]
float64 y # [m]
float64 theta # [rad]